typescript/vrotsc/src/compiler/program.ts

```diff
@@ -153,14 +153,15 @@ export function createProgram(options: ProgramOptions): Program {
 
 		filePaths.forEach(filePath => fileSet[filePath.toLowerCase()] = true);
 
-		if (options.files && options.files.length) {
+		if (options.files?.length) {
 			const filesFiltered = options.files.filter((item, i, ar) => ar.indexOf(item) === i);
 			const selectedFiles = [];
 			filesFiltered.forEach(fileName => {
-				for (let i = 0; i < filePaths.length; i++) {
-					const item = filePaths[i];
-					if (item.includes(fileName, 0))
+				for (const element of filePaths) {
```

Comment:
This will probably be even easier with a `forEach(element => ...`

Reply:
I think no longer applies after I removed the commit